src/test/java/nl/knaw/dans/pf/language/ddm/api/Ddm2EmdCrosswalkTest.java

```diff
@@ -127,6 +127,7 @@ private static void checkTestDataConsistency(File testFiles) {
             { "temporalABR" },
             { "temporalPlainText" },
             { "typeWithSchemeAndId" },
+            { "typeWithAndWithoutSchemeAndId" },
```

Comment:
Can't this method traverse the test dir for input files and return those?

https://github.com/DANS-KNAW/easy-ddm/blob/77ab3b3e0b92618b5a649a2753a7ff1d74cadbc5/src/test/java/nl/knaw/dans/pf/language/ddm/api/Ddm2EmdCrosswalkTest.java#L47

Reply:
It could, but it would require testing the test itself. Maybe that is why @rvanheest decided against this, when he refactored this test. Anyway, out of scope for this issue, I should say.

Reply:
It could, but it would require testing the test itself. Maybe that is why @rvanheest decided against this, when he refactored this test. Anyway, out of scope for this issue, I should say.

Reply:
It could, but it would require testing the test itself. Maybe that is why @rvanheest decided against this, when he refactored this test. Anyway, out of scope for this issue, I should say.

Reply:
It could, but it would require testing the test itself. Maybe that is why @rvanheest decided against this, when he refactored this test. Anyway, out of scope for this issue, I should say.

Reply:
It could, but it would require testing the test itself. Maybe that is why @rvanheest decided against this, when he refactored this test. Anyway, out of scope for this issue, I should say.